app/src/main/java/org/ieeeguc/ieeeguc/models/Committee.java

```diff
@@ -93,4 +97,45 @@ public void onResponse(Call call, Response response){
         });
 
     }
+    public void edit(int id , String token , String name , String description , final HTTPResponse HTTP_RESPONSE){
```

Comment:
How do you want the method without Id ? 
The method have ID in its header 
api/committee/{id} put